charts/jobs/templates/job.yaml

```diff
@@ -14,6 +14,11 @@ spec:
       annotations:
         checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") $ | sha256sum | trunc 63 }}
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") $ | sha256sum | trunc 63 }}
+        {{- if $spec.annotations }}
```

Comment:
With

Reply:
ok, I will change it everywhere and keep in mind the convention 👍

Reply:
The problem is that there is no standard convention and charts have the **if** version or the **with** version and I have also committed PR's to the originating chart repositories which lacked annotation support.